index.html

```diff
@@ -23,20 +23,22 @@
 <div class="top-w-2">
 <div id="main-nav">
 <ul>
-<li><a href="/quick-start">Quick Start</a></li>
-<li><a href="/integration-playbook">Integration Playbook</a></li>
+<li><a href="/getting-started">Quick Start</a></li>

```

Comment:
I notice on the main navigation that there is overflow of nav headers i.e. more nav headers than there should be or css is applied